docs/api-inference/index.md

```diff
@@ -1,61 +1,172 @@
-# Serverless Inference API
+# Inference Providers
 
-**Instant Access to thousands of ML Models for Fast Prototyping**
+The Hugging Face Inference Providers revolutionizes how developers access and run machine learning models by offering a unified, flexible interface to multiple serverless inference providers. This new approach extends our previous Serverless Inference API, providing more models, increased performances and better reliability thanks to our awesome partners.
```

Comment:
```suggestion
Hugging Face Inference Providers revolutionizes how developers access and run machine learning models by offering a unified, flexible interface to multiple serverless inference providers. This new approach extends our previous Serverless Inference API, providing more models, increased performances and better reliability thanks to our awesome partners.
```
(or `The Hugging Face Inference Providers API` like it seems it was at some point, not sure when it disappeared)